### ISSUE TITLE
Building for Windows on Linux: qtremoteobjects.index is missing in 5.8

### ISSUE THREAD

**Issue by quantonganh** [therecipe/qt#460]
https://github.com/therecipe/qt/issues/455#issuecomment-331335389

> build-ui-client-windows_1  |ESC[0m time="2017-09-22T02:36:41Z" level=debug msg="failed to load (optional) /home/user/work/src/github.com/therecipe/qt/internal/binding/files/docs/5.8.0/qtremoteobjects.index" error="open /home/user/work/src/github.com/therecipe/qt/internal/binding/files/docs/5.8.0/qtremoteobjects.index: no such file or directory"

It's here, in 5.9:

```
-rw-r--r-- 1 root root 523644 Sep 20 21:45 /home/user/work/src/github.com/therecipe/qt/internal/binding/files/docs/5.9.0/qtremoteobjects.index
```

**Comment by bclermont** [comment]
https://github.com/therecipe/qt/issues/455#issuecomment-330986504 it's not updated to 5.9?

**Comment by therecipe** [comment]
@quantonganh 
Yeah, that's because the "remoteobjects" pakcage is only available in Qt >= 5.9, but the windows images are still on 5.8. But it's just a warning, this shouldn't affect the build at all.

@bclermont 
Yes, mxe still only provides Qt 5.8 builds: http://pkg.mxe.cc/releases/
The latest prebuilt packages are from the "20-May-2017", so there should be a new release soon.

**Comment by quantonganh** [comment]
> Yeah, that's because the "remoteobjects" pakcage is only available in Qt >= 5.9, but the windows images are still on 5.8. But it's just a warning, this shouldn't affect the build at all.

The thing is the next error is: https://github.com/therecipe/qt/issues/461#issue-259691999

> Project ERROR: Unknown module(s) in QT: remoteobjects

Do you have any idea about this?
